pegasus/sites.v3/advocacy.code.org/views/forum_banner.haml

```diff
@@ -0,0 +1,12 @@
+=inline_css 'forum-banner.css'
+
+.announcement
+  .left
+    %p.title
+      You're invited to the 2018 Code.org/CSTA State Policy Forum
+    %p.details
+      September 27-28  Denver Colorado.
```

Comment:
tiny nit: Do we want "Denver, Colorado"? The comma is in the image with the city scape.

Reply:
Good call.